src/guides/components/findings.md

```diff
@@ -0,0 +1,65 @@
+# Using Findings
+
+Any requests or responses can be parsed for notable characteristics based on conditional statements using [Findings](https://docs.caido.io/guides/findings.html). As Caido proxies traffic, if it detects what you are looking for, an alert will be generated to draw your attention.
+
+## Creating Findings
+
+To create a Finding, use `sdk.findings.create()`. The `title`, `reporter` and `request` properties are required:
+
+``` ts
+await sdk.findings.create({
+  title: "Title", // Label your Finding.
+  description: "Description", // Add a description (optional).
+  reporter: "Reporter", // Specify which plugin discovered the Finding.
+  dedupe: `${request.getHost()}-${request.getPath()}`, // Prevents multiple alerts for request with matching characteristics (optional).
```

Comment:
dedupeKey

Reply:
Can you expand on the concept in an info box, I would like to explain to the user that they can craft they dedupeKey however they like and maybe give a few examples. It can contain anything, including response elements.